udata/templates/dataset/resource/list-item.html

```diff
@@ -41,7 +41,7 @@ <h4 class="list-group-item-heading ellipsis">
     {{ resource.owner|avatar(52, 'resource-owner') }}
     {% endif %}
 
-    {% if can_edit_resource(resource, dataset) %}
+    {% if can_edit_resource(resource if resource.from_community else dataset) %}
```

Comment:
Yes it does, but as discussed earlier, I made the minimal change to make it work until a proper cleanup and refactoring is done.